Source/Charts/Components/AxisBase.swift

```diff
@@ -200,15 +192,31 @@ open class AxisBase: ComponentBase
     /// Flag indicating that the axis-max value has been customized
     internal var _customAxisMax: Bool = false
     
-    /// Do not touch this directly, instead, use axisMinimum.
-    /// This is automatically calculated to represent the real min value,
-    /// and is used when calculating the effective minimum.
-    internal var _axisMinimum = Double(0)
+    /// The minimum value for this axis.
+    /// If set, this value will not be calculated automatically depending on the provided data.
+    /// Use `resetCustomAxisMin()` to undo this.
+    @objc open internal(set) var axisMinimum = 0.0 {
+        willSet {
+            _customAxisMax = true
```

Comment:
why willSet here? can't be in didSet?

Reply:
I was following the order in the former setters because I didn't want to change any logic in this PR. I agree it can all be done in didSet. Should I change it?

Reply:
I think so. The old setter is atomic I assume, so will/did seems the same in our case.